.github/workflows/generate-dto.yml

```
@@ -0,0 +1,45 @@
+            # -i bancos.json        -> Input file
+            # -o dist/bancos.ts     -> Output file
+            # --lang ts             -> Output language (TypeScript)
+            # --just-types          -> Generate only interfaces/types, without extra functions.
+            mkdir -p dist # Ensure the 'dist' folder exists
+            quicktype -i bancos.json -o dist/bancos.ts --lang ts --just-types
+            echo "DTO file generated at dist/bancos.ts"
+
+        - name: Commit generated files
+          uses: stefanzweifel/git-auto-commit-action@v5
+          with:
+            commit_message: 'chore(DTO): Update generated DTOs'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 14 but found 12

(indentation)